### DISCUSSION TITLE
Imposible abrir Firefly

### DISCUSSION THREAD

**Discussion by ROCKUMM** [iotaledger/firefly#8272] [Help & Support]
Quiero abrir Firefly  y se queda bloqueado con la frase "generando cuentas" y la mosca aleteando... Imposible abrir la wallet, ni ver mi cuenta. 
I want to open firefly and it stays blocked with the phrase "generating accounts" and the fly flapping... Impossible to open the wallet.

**Comment by begonaalvarezd** [comment]
Si estas en ledger, necesitas actualizar tu iota ledger app a la version compatible 0.8.6, si no sale en ledger live, significa que tienes que actualizar el firmware primero. Cuando tengas la ledger actualizada, conectala y ten la app abierta a ver si así puede por fin generarte las wallets

**Comment by ROCKUMM** [comment]
Y cómo actualizo el firmware?

**Comment by ROCKUMM** [comment]
Ya actualizado a la versión 0.8.6.
Y nada, sigue "aleteando", pero no se puede abrir...

**Reply by begonaalvarezd** [reply]
Asegurate de estar usando un cable original de ledger, ledger da problemas cuando el cable no es original
De momento no se me ocurre otra cosa que pueda estar pasando, para continuar con support te recomiendo que vayas a nuestro discord https://discord.iota.org/ puedes abrir un ticket de servicio en el canal `#help`

**Comment by ROCKUMM** [comment]
Uso el cable original.
Gracias.
